homeassistant/components/ozw/climate.py

```diff
@@ -274,8 +277,14 @@ async def async_set_fan_mode(self, fan_mode):
 
     async def async_set_hvac_mode(self, hvac_mode):
         """Set new target hvac mode."""
+        if not self.values.mode:
+            # Thermostat(valve) with no support for setting a mode
+            _LOGGER.warning(
+                "Thermostat %s does not support setting a mode", self.entity_id
+            )
+            return HVAC_MODE_HEAT
```

Comment:
This method should not return anything. Just `return`.

Reply:
Oops, how sloppy of me. Will fix